lms/djangoapps/discussion_api/serializers.py

```diff
@@ -290,6 +290,15 @@ class CommentSerializer(_ContentSerializer):
 
     non_updatable_fields = NON_UPDATABLE_COMMENT_FIELDS
 
+    def __init__(self, *args, **kwargs):
+        remove_fields = kwargs.pop('remove_fields', None)
```

Comment:
I thought to keep it consistent with how the arguments are being passed. Does it not make sense? and if we have better understanding to explicitly adding `remove_fields` argument; I can change to it too.